docs/_pages/releases.md

```diff
@@ -10,6 +10,11 @@ sidebar:
 ## Unreleased
 
 ### What's New
+* Annotated `[Not]MatchRegex(string)` with `[StringSyntax("Regex")]` which IDEs can use to colorize the regular expression argument - [#1816](https://github.com/fluentassertions/fluentassertions/pull/1816)
```

Comment:
🤔 What happened here?

Reply:
I'm really sorry about the mess, Dennis! My fork's default branch was inadvertently set to `master` rather than `develop`. Therefore my attemps to fix a merge conflict resulted in this fuss. I rebased the PR onto the latest `develop` and now it is looking better.